### PR TITLE
fix: package dot sh and chart.yaml versions 

### PR DESCRIPTION
**Pull request by tdaly61**
Hi @mdebarros : per my long comments in slack and on the other 2 recent PR's this PR when merged will fix the package.sh and update-chart-dep.sh to include the correct Bof charts and the correct version (2.0.0) for the common chart and thus fix the circle-ci pipeline issues.  **Bottom line** , if you review and approve this PR .I will merge and .. you can (I am confident) ignore the previous conversations about why the pipleine fails , because it will be fixed :-)

### CODE REVIEW

File: scripts/update-charts-dep.sh [hunk 1]
```diff
@@ -28,11 +30,15 @@ declare -a charts=(
         mojaloop/reporting-hub-bop-trx-ui
         mojaloop/reporting-legacy-api
         mojaloop/security-hub-bop-kratos-ui
+        #mojaloop/security-role-perm-crd
```
Comment (mdebarros) [inline scripts/update-charts-dep.sh:33]: This does not exist anymore, so I would recommend removing it.

**Review by mdebarros — CHANGES_REQUESTED** [review]
Looks good, except for the unnecessary commented out line in the update script.

File: scripts/update-charts-dep.sh (hunk 1 above)
Comment (tdaly61) [reply scripts/update-charts-dep.sh:33]: will do

Comment (tdaly61) [reply scripts/update-charts-dep.sh]: done !

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**